lib/apr/views/seller_slack_view.ex

```diff
@@ -21,6 +24,34 @@ defmodule Apr.Views.InvoiceSlackView do
     @gravity_api.get!("/partners/#{partner_id}").body
   end
 
+  defp external_account_restricted_soon_message(event, partner_data) do
+    {:ok, due_date} = DateTime.from_unix(event["properties"]["stripe_requirements"]["deadline"])
+    %{
+      text: ":warning: Stripe account of #{partner_data["name"]} will be restricted by #{NimbleStrftime.format(due_date, "%Y-%m-%d %I:%M %p %Z")}",
```

Comment:
Note to self: The `strftime/3` has been introduced to `Calendar` since Elixir v 1.11.0. https://hexdocs.pm/elixir/1.11.0/Calendar.html#strftime/3

Reply:
Non-blocking. We can ask feedback about the exact copy. Presumably we can call out the action to check on Stripe dashboard (by following the link below the message).

Reply:
> We can ask feedback about the exact copy.

I'd like to show Andrew Boos what the exact message looks like on Slack, so I'll deploy this code to the staging and then ask for feedback. 

> Presumably we can call out the action to check on Stripe dashboard (by following the link below the message).

Do you mean that it's better to change the label of the `Stripe account ID` link to the Stripe dashboard to `Please check why the account will be restricted soon on the Stripe dashboard`?